packages/svelte/src/compiler/types/index.d.ts

```diff
@@ -241,7 +241,8 @@ export interface Binding {
 	node: Identifier;
 	/**
 	 * - `normal`: A variable that is not in any way special
-	 * - `prop`: A normal prop (possibly mutated)
+	 * - `prop`: A normal prop (possibly reassigned)
```

Comment:
this change was from when there was a compiler error, which is no longer true
```suggestion
	 * - `prop`: A normal prop (possibly reassigned or mutated)
```